src/engine/renderer/tr_backend.cpp

```diff
@@ -5604,17 +5604,31 @@ const RenderCommand *FinalisePortalCommand::ExecuteSelf( ) const
 
 	GL_LoadModelViewMatrix( backEnd.orientation.modelViewMatrix );
 
+	GL_State( GLS_DEPTHMASK_TRUE | GLS_DEPTHFUNC_ALWAYS );
+	glState.glStateBitsMask = GLS_DEPTHMASK_TRUE | GLS_DEPTHFUNC_ALWAYS;
+
+	glStencilFunc( GL_EQUAL, backEnd.viewParms.portalLevel + 1, 0xff );
+	glStencilOp( GL_KEEP, GL_KEEP, GL_KEEP );
+
+	Tess_Begin( Tess_StageIteratorGeneric, nullptr, shader,
+		nullptr, false, false, -1, -1 );
+	rb_surfaceTable[Util::ordinal( *( surface->surface ) )]( surface->surface );
+	Tess_End();
+
 	glStencilFunc( GL_EQUAL, backEnd.viewParms.portalLevel + 1, 0xff );
```

Comment:
The stencil calls now appear twice. Is that intentional?

Reply:
You mean the glStencilFunc() call? One of those can be removed, good call

Reply:
Assuming the above, fixed